src/main/java/de/uhd/ifi/se/decision/management/jira/extraction/versioncontrol/GitRepositoryFSManager.java

```diff
@@ -69,36 +52,32 @@ public GitRepositoryFSManager(String home, String project, List<String> repoUris
 	 *            branch name, Repository Uri for branch
 	 * @return null on failure, absolute path to new temporary directory
 	 */
-	public String releaseBranchDirectoryNameToTemp(String branchShortName, String repoUri) {
-		if (baseProjectUriPaths.containsKey(repoUri)) {
-			String checkoutPath = getCheckoutPath(branchShortName, repoUri);
-			if (!"".equals(checkoutPath)) {
-				File oldDir = new File(getCheckoutPath(branchShortName, repoUri));
-				if (!oldDir.isDirectory()) {
-					return null;
-				} else {
-					Date date = new Date();
-					long time = date.getTime();
-					String tempDirString = baseProjectUriPaths.get(repoUri) + File.separator + TEMP_DIR_PREFIX + time;
-					File tempDir = new File(tempDirString);
-					boolean renameResult = false;
-					try {
-						renameResult = oldDir.renameTo(tempDir);
-					} catch (Exception e) {
-						LOGGER.error("Could not rename " + oldDir + " to " + tempDirString + ". " + e.getMessage());
-						return null;
-					}
-					if (!renameResult) {
-						LOGGER.error(
-								"Could not rename " + oldDir + " to " + tempDirString + ". The reason is not known.");
-						return null;
-					}
-					removeBranchPathMarker(branchShortName, repoUri);
-					return tempDirString;
-				}
-			}
+	public String releaseBranchDirectoryNameToTemp(String branchShortName) {
+		String checkoutPath = getCheckoutPath(branchShortName);
+		if (checkoutPath.isBlank()) {
+			return null;
```

Comment:
Auch hier macht eine Empty-String deutlich mehr Sinn als null, so das man in Aufrufenden Methoden auf isEmpty() prüfen kann